sdk/openai/ai-openai/samples-dev/listChatCompletions.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Demonstrates how to list chat completions for a chat context.
+ *
+ * @summary list chat completions.
+ * @azsdk-weight 100
```

Comment:
We may want to change the weight value to give slightly more preference to SSE vs non-SSE operations